src/release/mod.rs

```diff
@@ -324,8 +312,7 @@ impl DaemonRuntime {
     ) -> RelResult<()> {
         self.terminate_background_applications();
 
-        // Check the system and perform any repairs necessary for success.
-        repair::repair().map_err(ReleaseError::Repair)?;
+        let from_codename = from.parse::<Codename>().unwrap();
```

Comment:
Failing on [this line](https://github.com/pop-os/upgrade/blob/da0d8153e879c73e89ec674f06b973bf933a4c55/src/release/mod.rs#L315):

```
Oct 19 15:08:36 pop-os pop-upgrade[7535]: killing io.elementary.appcenter
Oct 19 15:08:36 pop-os pop-upgrade[7535]: thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: NotFound', src/release/mod.rs:315:29
Oct 19 15:08:36 pop-os pop-upgrade[7535]: note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

The backtrace doesn't look too useful, but here it is: [backtrace.txt](https://github.com/pop-os/upgrade/files/5404795/backtrace.txt)

Reply:
Oops. I know what happened